commit fd4269c99f9d6f3a52bac0013fe8dbf0ac3f9df7
Author: Dawit Alemayehu <adawit@kde.org>
Date:   Sun Oct 6 09:32:22 2013 -0400

    Fixed missing signal/slot connection:
    
    Object::connect: No such slot KDEBrowser::teardown()
    Object::connect:  (sender name:   'Bookmarks')
    
    REVIEW: 113131

diff --git a/plasma/generic/runners/bookmarks/bookmarksrunner.cpp b/plasma/generic/runners/bookmarks/bookmarksrunner.cpp
index b36ad7f..45093af 100644
--- a/plasma/generic/runners/bookmarks/bookmarksrunner.cpp
+++ b/plasma/generic/runners/bookmarks/bookmarksrunner.cpp
@@ -57,7 +57,7 @@ void BookmarksRunner::prep()
 {
     m_browser = m_browserFactory->find(findBrowserName(), this);
     connect(this, SIGNAL(teardown()), dynamic_cast<QObject*>(m_browser), SLOT(teardown()));
-	m_browser->prepare();
+    m_browser->prepare();
 }
 
 
diff --git a/plasma/generic/runners/bookmarks/browsers/kdebrowser.h b/plasma/generic/runners/bookmarks/browsers/kdebrowser.h
index 3302e41..c23f361 100644
--- a/plasma/generic/runners/bookmarks/browsers/kdebrowser.h
+++ b/plasma/generic/runners/bookmarks/browsers/kdebrowser.h
@@ -40,6 +40,10 @@ class KDEBrowser : public QObject, public Browser
 public:
     explicit KDEBrowser(QObject *parent = 0);
     virtual QList<BookmarkMatch> match(const QString& term, bool addEverything);
+
+public Q_SLOTS:
+    virtual void teardown() {}
+
 private:
     KBookmarkManager * const m_bookmarkManager;
     Favicon * const m_favicon;
